src/station/station_info.rs

```diff
@@ -0,0 +1,946 @@
+    ///  Last unicast data frame receive rate, nested attribute, like
+    /// [`NL80211_STA_INFO_TX_BITRATE`].
+    RxBitrate(Vec<Nl80211RateInfo>),
+    /// Total transmitted packets (MSDUs and MMPDUs)
+    TxPackets(u32),
+    /// Total received packets (MSDUs and MMPDUs)
+    RxPackets(u32),
+    /// Total number of received packets (MPDUs)
+    RxMpdus(u32),
+    /// Total retries (MPDUs)
+    TxRetries(u32),
+    /// Total failed packets (MPDUs) (u32, to this station)
+    TxFailed(u32),
+    /// Transmitted packets dropped for unspecified reasons
+    RxDropMisc(u64),
+    /// Total number of packets (MPDUs) received with an FCS error . This count
+    /// may not include some packets with an FCS error due to TA corruption.
+    /// Hence this counter might not be fully accurate.
+    FcsErrorCount(u32),
+    /// The station's mesh LLID
+    Llid(u16),
+    /// The station's mesh PLID
+    Plid(u16),
+    /// Peer link state for the station (see [`Nl80211PeerLinkState`])
+    PeerLinkState(Nl80211PeerLinkState),
+    /// Current station's view of BSS, nested attribute containing info as
+    /// possible, see [`Nl80211StationBssParam`]
+    BssParam(Vec<Nl80211StationBssParam>),
+    /// Time since the station is last connected
+    ConnectedTime(u32),
+    /// Contains a [Nl80211StationFlagUpdate]
+    StationFlags(Nl80211StationFlagUpdate),
+    /// Timing offset with respect to this station
+    TimingOffset(i64),
+    /// Local mesh staion link-specific power mode
+    LocalPowerMode(Nl80211MeshPowerMode),
+    /// Peer mesh staion link-specific power mode
+    PeerPowerMode(Nl80211MeshPowerMode),
+    /// Neighbor mesh station power save mode towards non-peer station
+    NonPeerPowerMode(Nl80211MeshPowerMode),
+    /// Per-chain signal strength of last PPDU. Contains a nested array of
+    /// signal strength attributes (dBm)
+    ChainSignal(Vec<u8>),
+    /// Per-chain signal strength average. Same format as
+    /// [`Nl80211StationInfo::ChainSignal`]
+    ChainSignalAvg(Vec<u8>),
+    /// Expected throughput considering also the 802.11 header (kbps)
+    ExpectedThroughput(u32),
+    /// Number of beacons received from this peer
+    BeaconRx(u64),
+    /// Signal strength average for beacons only (dBm)
+    BeaconSignalAvg(u8),
+    /// Count of times beacon loss was detected
+    BeaconLoss(u32),
+    /// This is a nested attribute where each the inner attribute number is the
+    /// TID+1 and the special TID 16 (i.e. value 17) is used for non-QoS
+    /// frames; each one of those is again nested with &enum nl80211_tid_stats
+    /// attributes carrying the actual values.
+    TidStats(Vec<NestedNl80211TidStats>),
+    /// Aggregate PPDU duration for all frames sent to the station (usec)
+    TxDuration(u64),
+    /// Aggregate PPDU duration for all frames received from the station (usec)
+    RxDuration(u64),
+    /// Attribute used for padding for 64-bit alignment
+    /// TODO: Verify behaviour
+    Pad,
```

Comment:
Pad is handled by netlink-packet-core code, you will never hit it at this level.